client/components/App.jsx

```diff
@@ -12,13 +13,17 @@ const StyledWrapper = styled.div`
   max-width: 1024px;
 `;
 
-const App = ({ match }) => (
-  <StyledWrapper>
-    <Title />
-    <ReviewSummary product_id={Number(match.params.id)} />
-    <SearchReviews product_id={Number(match.params.id)} />
-    <Reviews product_id={Number(match.params.id)} />
-  </StyledWrapper>
-);
+const App = ({ match }) => {
+  const reviewSummaryData = useAPI(`http://localhost:3001/api/reviews/${match.params.id}/summary`);
```

Comment:
I like the use of React Hooks. I might have to try this on mine instead of Fetch.